src/components/SelectionList/UserListItem.tsx

```diff
@@ -62,6 +72,27 @@ function UserListItem({
         >
             {(hovered?: boolean) => (
                 <>
+                    {canSelectMultiple && checkmarkPosition === CONST.DIRECTION.LEFT && (
```

Comment:
one of the reasons of this refactoring was to get rid of this unnecessary `checkmarkPosition` prop - instead I would keep this block for `UserListItem` where the checkbox is shown on the left of the item, and create completely new list item component to handle items with the checkmark on the right

Reply:
Sorry - I was busy working on another issue. As this list item will be used only on the Invite Member page, maybe `InviteMemberListItem`? What do you think?